tests/components/group/test_cover.py

```diff
@@ -209,6 +210,59 @@
                .attributes.get(ATTR_CURRENT_POSITION) == 0
 
 
+async def test_toggle_covers(hass, setup_comp):
+    """Test toggle cover function."""
+    # Start covers in open state
+    await hass.services.async_call(
+        DOMAIN, SERVICE_OPEN_COVER,
+        {ATTR_ENTITY_ID: COVER_GROUP}, blocking=True)
+    for _ in range(10):
+        future = dt_util.utcnow() + timedelta(seconds=1)
+        async_fire_time_changed(hass, future)
+        await hass.async_block_till_done()
+
+    state = hass.states.get(COVER_GROUP)
+    assert state.state == STATE_OPEN
+
+    # Toggle will close covers
+    await hass.services.async_call(
+        DOMAIN, SERVICE_TOGGLE,
+        {ATTR_ENTITY_ID: COVER_GROUP}, blocking=True)
+    for _ in range(10):
+        future = dt_util.utcnow() + timedelta(seconds=1)
+        async_fire_time_changed(hass, future)
+        await hass.async_block_till_done()
+
+    state = hass.states.get(COVER_GROUP)
+    assert state.state == STATE_CLOSED
+    assert state.attributes.get(ATTR_CURRENT_POSITION) == 0
```

Comment:
It looks like the attributes aren't required in other tests, e.g. `assert state.attributes.get(ATTR_CURRENT_POSITION) is None`

Reply:
Right, only change if you expect the attribute be present.  The example provided is a bad assertion test...   Is it testing that the value of `ATTR_CURRENT_POSITION` is `None` or is it testing that the attribute is not present in the dictionary?  We don't know because it will pass on both conditions.

If you want to test that the attribute is not present, it should be asserted this way: `assert ATTR_CURRENT_POSITION not in state.attributes`.

Reply:
Hm ok, looks like all the existing tests have asserts that need to be updated then